pkg/trebuchet/types.go

```diff
@@ -0,0 +1,14 @@
+package trebuchet
+
+import (
+	"net/http"
+
+	"github.com/atlassian/voyager/pkg/util/pkiutil"
+	"go.uber.org/zap"
+)
+
+type ExtraConfig struct {
```

Comment:
This should be constructed via `ApplyTo` method, currently isn't AFAICT.
See https://github.com/atlassian/voyager/blob/master/pkg/creator/server/options/creator_options.go#L35-L59